src/store/action-types.ts

```diff
@@ -1,3 +1,3 @@
-export const WEATHER_CITY = "WEATHER_CITY";
-export const WEATHER_DAYS = "WEATHER_DAYS";
-export const SET_COUNTS = "SET_COUNTS";
+export const WEATHER_CITY = 'WEATHER_CITY';
+export const WEATHER_ALL = 'WEATHER_ALL';
```

Comment:
maybe GET_WHEATHER_ALL. it's a description of an action. and the action is doing something

Reply:
done